Readme.md

```diff
@@ -18,171 +18,444 @@ This is an ipm package, which contains one or more reusable assets within the ip
 
 ## Setup
 
-_Add any setup instructions, such as an API Key_
+API key can be generated/accessed by going to this page: https://cloud.google.com/bigquery/docs/authorization
+ 
+Another simple way: Copy and paste the following command 'gcloud auth application-default print-access-token' in Google Cloud Platform shell!
```

Comment:
preface with: 'If you already have a GCloud account and the CLI set up," ...

Reply:
Done